templates/gitea.ini.j2

```diff
@@ -1,3 +1,10 @@
+; this file is the configuration of your local gitea instance
```

Comment:
Use `; {{ ansible_managed }}` instead of this spiel

Reply:
You are right, i forgot that.

It is now added!